db/migrate/20180924185921_devise_token_auth_create_users.rb

```diff
@@ -0,0 +1,58 @@
+class DeviseTokenAuthCreateUsers < ActiveRecord::Migration[5.2]
+  def change
+
+    #drop_table :users
+    #create_table(:users) do |t|
+      ## Required
+    add_column :users, :provider, :string, :null => false, :default => "email"
+    add_column :users, :uid, :string, :null => false, :default => ""
+
+      ## Database authenticatable
+      #add_column :users, :encrypted_password, :string, :null => false, :default => ""
+
+      ## Recoverable
+      #add_column :users, :reset_password_token, :string
+      #add_column :users, :reset_password_sent_at, :datetime
+    add_column :users, :allow_password_change, :boolean, :default => false
+
+      ## Rememberable
+      #add_column :users, :remember_created_at, :datetime
+
+      ## Trackable
```

Comment:
In your model you are configuring only these option for devise user:
` database_authenticatable, :registerable, :rememberable, :validatable`

Maybe you are doing some migration useless, no?

Reply:
I forgot to remove them when I updated the user model. I will update it now.